AmplifyTestCommon/Helpers/AuthSignInHelper.swift

```diff
@@ -6,7 +6,6 @@
 //
 
 import Amplify
-import XCTest
```

Comment:
this is needed because the currently we're adding the AmplifyTestCommon files to the host app target directly to provide access to the tests to use. These imports were not used and cannot be imported when adding to an app target